generated/nidmm/setup.py

```diff
@@ -28,7 +28,7 @@ def read_contents(file_to_read):
 setup(
     name=pypi_name,
     zip_safe=False,
-    version='0.1.0.dev3',
+    version='0.1.0.dev4',
```

Comment:
Was this a manual change?
Seems unrelated to PR.

Reply:
I build .dev3 and pushed to PyPI. This updates the config['version'] but that hadn't been built into the generated files yet.